components/form/demo/control-hooks.md

```
@@ -2,10 +2,10 @@
 
 通过 `Form.useForm` 对表单数据域进行交互。
 
-> 注意 `useForm` 是 [React Hooks](https://reactjs.org/docs/hooks-intro.html) 的实现，只能用于函数组件。如果是在 Class Component 下，你也可以通过 `ref` 获取数据域：https://codesandbox.io/p/sandbox/ngtjtm
+> 注意 `useForm` 是 [React Hooks](https://zh-hans.react.dev/reference/react/hooksl) 的实现，只能用于函数组件。如果是在 Class Component 下，你也可以通过 `ref` 获取数据域：https://codesandbox.io/p/sandbox/ngtjtm
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)